src/raftstore/store/config.rs

```diff
@@ -28,8 +28,8 @@ const SPLIT_REGION_CHECK_TICK_INTERVAL: u64 = 10000;
 const REGION_SPLIT_SIZE: u64 = 64 * 1024 * 1024;
 const REGION_MAX_SIZE: u64 = 80 * 1024 * 1024;
 const REGION_CHECK_DIFF: u64 = 8 * 1024 * 1024;
-const REGION_COMPACT_CHECK_TICK_INTERVAL: u64 = 300_000;
-const REGION_COMPACT_DELETE_KEYS_COUNT: u64 = 1_000_000;
+const REGION_COMPACT_CHECK_TICK_INTERVAL: u64 = 300;
+const REGION_COMPACT_DELETE_KEYS_COUNT: u64 = 200_000;
```

Comment:
I think exporting these and use them directly in tikv-server is better to avoid default configurations mismatched.